### PR TITLE
draft: resync document

### PR DESCRIPTION
**Pull request by Tal-or**
This document aim to captures the way of maintaining resync between scheduler-plugins u/s and this repo

Signed-off-by: Talor Itzhak <titzhak@redhat.com>

### CODE REVIEW

File: RESYNC.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+## Creating a new local branch for the new resync
+
+Branch the target openshift-kni/scheduler master branch to a new resync local branch 
+
+`git checkout master`
+
+`git checkout -b "resync-$(date +%Y%m%d)"`
+
+## Merge changes from upstream
```
Comment (cynepco3hahue) [inline RESYNC.md:27]: it better to work via cherry-picking than merge the whole branch.

Comment (ffromani) [reply RESYNC.md:27]: Do we have options to automate the cherry-pick of a massive set of changes?
While I agree about the merits of cherry-pick (I'm open to this option if we can make it manageable and not manual), this repo must always be merge-compatible. IOW, even if we decide to not actually do it, it should always be _possible_ to merge u/s changes. To ensure this we need to strive to make compatible, mostly additive changes

**Review by ffromani — COMMENTED** [review]
LGTM pending the merge vs cherry-pick approach

File: RESYNC.md (hunk 1 above)
Comment (Tal-or) [reply RESYNC.md:27]: > (I'm open to this option if we can make it manageable and not manual)

merge everything, then rebase and delete the undesired commits, but the rebase will override the tree and you'll get a different commit hashes, so I don't think we want that.

Comment (Tal-or) [reply RESYNC.md:27]: @cynepco3hahue @fromanirh Do I have a green light to push this forward?
I need your help with moving this repo (once I'm done) under the openshift-kni org and archiving the old one

Comment (ffromani) [reply RESYNC.md:27]: I'm ok with the policy described in the doc.

**Comment by ffromani** [comment]
> LGTM pending the merge vs cherry-pick approach

I'm inclined towards the approach described in the doc but I'm open to a cherry-pick flow if properly discussed and after evaluation of pros/cons
so LGTM for the current state of things.

**Pull request closed without merging**